examples/opf/experiments/classification/base_category/description.py

```diff
@@ -187,9 +187,9 @@
             # is correct here as opposed to 'columns')
             'synPermConnected': 0.1,
 
-            'synPermActiveInc': 0.1,
+            'synPermActiveInc': 0.05,
 
-            'synPermInactiveDec': 0.01,
+            'synPermInactiveDec': 0.008,
```

Comment:
Yes these were the only values I could find that allowed the integration tests to pass. I think changing the defaults makes sense - @subutai should I go ahead and do that?